skills/spry-docs/SKILL.md

```
@@ -0,0 +1,105 @@
+metadata:
+  version: "1.0"
+  package: spry
+---
+
+# Spry Framework Guide
+
+Spry is a Dart server framework centered on **filesystem routing** and **generated runtime output**. The filesystem is the source of truth for route structure — everything is explicit and inspectable.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)